### PR TITLE
BBD-237: SAYT preview searches for all suggestion types

### PR DESCRIPTION
**Pull request by effervescentia**

### CODE REVIEW

File: test/unit/tags/sayt.ts [hunk 1]
```diff
@@ -216,10 +241,31 @@ suite('gb-sayt', Sayt, ({
...
+      const searchProducts = stub(tag(), 'searchProducts');
+      tag().rewriteQuery = () => null;
+      tag()._config = { autoSearch: true, categoryField: 'size' };
+
+      tag().notifier(query, 'Medium');
+
+      expect(searchProducts.calledWith(query, '~size=Medium')).to.be.true;
+    });
+
+    it('should fetch suggestions with refinements and overrite query', () => {
```
Comment (Vic-Dev) [inline test/unit/tags/sayt.ts]: overwrite :)

**Review by Vic-Dev — APPROVED** [review]
Other than that looks good :)

File: test/unit/tags/sayt.ts (hunk 1 above)
Comment (effervescentia) [reply test/unit/tags/sayt.ts]: ah ty ty

**Review by Vic-Dev — APPROVED** [review]
👊

**Pull request merged**